packages/js/components/src/tag/index.tsx

```diff
@@ -16,15 +16,27 @@ import { withInstanceId } from '@wordpress/compose';
  *
  * @param {Object}        props
```

Comment:
These jsdocs aren't very useful after having type annotations - if you have more insight into the usage of these props, a better place would be to have them as inline comments above each key in the Props type object. Otherwise I think they should be removed?